tests/mtest/src/main/scala/tests/Assertions.scala

```diff
@@ -0,0 +1,70 @@
+package tests
+
+import munit.Location
+import scala.meta.io.AbsolutePath
+
+trait Assertions extends munit.Assertions {
+
+  def assertNoDiffOrPrintObtained(
+      obtained: String,
+      expected: String
+  )(implicit loc: Location): Unit = {
+    // FIXME(gabro): surface this in funsuite Assertions
+    munit.Diffs.assertNoDiff(obtained, expected)
```

Comment:
Does this print the obtained when it fails?

Reply:
Yes. That’s the default behavior

Reply:
cool, than we can just remove this method and use `assertDiff` directly